src/sagas.js

```diff
@@ -1,34 +1,82 @@
+import { loadModules } from '@esri/react-arcgis';
+import { esriGeometryToGeojson } from 'utils/geojson-parser';
 import { all, takeLatest, call, select, put, cancelled } from 'redux-saga/effects'
 import { reduxConfig as speciesModule } from 'redux_modules/species';
+import geoDescriberActions from 'redux_modules/geo-description';
 import axios from 'axios';
 
-const SPECIES_FETCH_DATA_READY = speciesModule.actions.SPECIES_FETCH_DATA_READY
-const SPECIES_FETCH_DATA_LOADING = speciesModule.actions.SPECIES_FETCH_DATA_LOADING
-const SPECIES_FETCH_DATA_ERROR = speciesModule.actions.SPECIES_FETCH_DATA_ERROR
+const { REACT_APP_GEO_DESCRIBER_API } = process.env;
+// const SPECIES_FETCH_DATA_READY = speciesModule.actions.SPECIES_FETCH_DATA_READY
+// const SPECIES_FETCH_DATA_LOADING = speciesModule.actions.SPECIES_FETCH_DATA_LOADING
+// const SPECIES_FETCH_DATA_ERROR = speciesModule.actions.SPECIES_FETCH_DATA_ERROR
 
-const molAPI = 'https://api.mol.org/1.x/species/info?scientificname='
+const SET_GEO_DESCRIPTION_LOADING = geoDescriberActions.setGeoDescriptionLoading
+const SET_GEO_DESCRIPTION_READY = geoDescriberActions.setGeoDescriptionReady
+const SET_GEO_DESCRIPTION_ERROR = geoDescriberActions.setGeoDescriptionError
 
-function* watchGridCellDataFetch() {
-  yield takeLatest('SET_GRID_CELL_DATA', fetchSpeciesData)
+// const molAPI = 'https://api.mol.org/1.x/species/info?scientificname='
```

Comment:
I assume that we want to keep this commented code here?

Reply:
yeah, is there to eventually drive the Species widget (that needs some rethinking)